src/facebook-mobile-preview/facebook-mobile-preview.ejs

```diff
@@ -255,6 +255,10 @@
       line-height: 20px;
     }
 
+    .-theme-with-dark-background {
+      color: var(--base-color);
```

Comment:
Should there not be a fallback color, for the case css variables are not supported?

Reply:
I think that kills the idea of having flexible variable which we can change in the future. 
you mean like this right?:
`.-theme-with-dark-background {
      color:#fff;
      color: var(--base-color);
    }`